src/core/css/inc/vars.scss

```diff
@@ -48,3 +48,6 @@ $max-modal-width: 768px;
 // Theme image dimensions
 $theme-height-default: 92px;
 $theme-width-default: 680px;
+
+// Lightweight images are slightly different
+$theme-height-legacy: 100px;
```

Comment:
We can't use `$theme-height-default` because legacy themes have a height of `100px`? I don't understand the comment above, so I am guessing :p

Reply:
oops :) Sorry my comment wasn't clear but yes, that's right. I am using the `100px` bc lightweight themes still have this.